crossfilter.js

```diff
@@ -582,6 +582,15 @@ crossfilter_bitarray.prototype.zeroExcept = function(n, offset, zero) {
   return true;
 };
 
+// Checks that all bits for the given indez are 0 except for the specified mask.
+// The mask should be an array of the same size as the filter subarrays width.
+crossfilter_bitarray.prototype.zeroExceptMask = function(n, mask) {
+  var i, len;
+  for (i = 0, len = this.subarrays; i < len; ++i)
+    if (this[i][n] & mask[i]) return false;
```

Comment:
Can you add braces to the `if` statement to align with the coding style in the rest of the file?